### PR TITLE
[Docs] Migrate Time API

### PR DESCRIPTION
**Pull request by Chocolate-lightning**

### CODE REVIEW

**Comment by andrewnicols** [comment]
Did you forget to check in the migrated page? I only see the migratedPages.yml file, and the fixed references to it.

**Comment by Chocolate-lightning** [comment]
Heh, oh my, that is a bit embarrassing. It seems my brain fell out on the floor as I was committing the changes. Checking now I have attached the file...

File: docs/apis/subsystems/time/index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+DST is abbreviation of **Daylight Saving Time** (also known as "Day light saving" and "Summer Time"). Many countries, and sometimes just certain regions of countries, adopt daylight saving time during part of the year. Moodle automatically calculates DST for current user, depending on user location.
+
+## Examples
+
+### Create DateTime with date/time from a unixtime (number of seconds)
+
+$date = new DateTime();
+$date->setTimestamp(intval($unixtime);
+echo userdate($date->getTimestamp());
```
Comment (andrewnicols) [inline docs/apis/subsystems/time/index.md:99]:
```suggestion
```php
$date = new DateTime();
$date->setTimestamp(intval($unixtime);
echo userdate($date->getTimestamp());
```
```

**Review by andrewnicols — CHANGES_REQUESTED** [review]
A couple of minor things and this is good to go.

**Review by andrewnicols — APPROVED** [review]

**Pull request merged**